cmd/dep/testdata/harness_tests/init/gb/case1/initial/src/gbtest/main.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	cdt "github.com/carolynvs/deptest"
```

Comment:
You don't have to change what you have here. I just wanted to share another pattern for making tests for dep:

```golang
package main

import (
  _ "github.com/carolynvs/deptest"
  _ "github.com/sdboyer/deptest"
)

func main() {}
```